ui/controls.js

```diff
@@ -660,6 +668,20 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
     }
   }
 
+  /** @export */
+  unload() {
+    const videoBar = document.getElementById('video-bar');
```

Comment:
This is specific to the demo page, and can't be applied generally to all apps.

Perhaps instead of manipulating parts of the DOM outside of the UI library's control, this should unload the video player and dispatch an event that the application (the demo, here) can use to make app-level DOM changes.

Reply:
Ohhh, yeah I'll do that